tools/ci/choose_workflows.py

```diff
@@ -56,7 +56,7 @@ def should_run_integration_tests(ctx: CIContext) -> bool:
         build_env=context.build_env.DD_OVERRIDE_RUN_INTEGRATION_TESTS,
         pr_keyword='[x] Run integration tests',
         pr_path_prefixes=[
-            'IntegrationTests/IntegrationTests/',
+            'IntegrationTests/',
```

Comment:
Previous path was wrong and integration tests weren't triggered automatically (required selecting checkbox in PR description). Now this is fixed.

Reply:
Great catch!